### DISCUSSION TITLE
Some Questions about Authhentication

### DISCUSSION THREAD

**Discussion by benpallen** [supabase/supabase#15736] [Questions]
I use supabase in a Quasar Vue app.

When I go to another browser tab from my app's tab on any browser and come back to my app's tab,  "SIGNED_IN" event triggers every time.

  supabase.auth.onAuthStateChange(async (event, session) => {
    console.log(event, session);
  }

Is this normal?    

The other issue is that when I close the browser and open it again and when I go to my app, it automatically logs me in.
It triggers "INITIAL_SESSION" event and the session is not null. 

Why is the session not null?

Maybe I do something wrong.
Is there anything I am supposed to do?

Thanks

**Comment by GaryAustin1** [comment]
Yes the event is normal for changing tabs.  It is not making API calls though.

Normally the clients store the session in local storage and will restore and refresh the session if you have not signed out.  I do not know Quasar Vue though so not sure if it has local storage, if not then something else is storing the session.

**Comment by rcourtman** [comment]
Disabling temperature monitoring does prevent the initial ssh-keyscan, but commit 8f0b07ee1 (2025-11-05) added per-node temperature toggles—use that instead of shutting the entire feature off during install. Check intervals aren't configurable; temperature collection runs on the same cadence as the main monitor loop by design so node snapshots stay in sync.

A sustained flood means the host keys aren't sticking. They're stored under `~/.ssh/known_hosts_sensors`. If that file keeps disappearing you're losing the container's home directory—wrong volume mount, read-only bind, or the container is being recreated without the `pulse-data` volume. Fix the persistence so the cache survives restarts.

Long term you should switch to `pulse-sensor-proxy`. Commit d271f79f0 shipped the multi-arch builds (Linux amd64/arm64/armv7). When the proxy socket is mounted Pulse never falls back to SSH, so there's no keyscan burst or root key material in the container.
